### PR TITLE
refactor: make privateArgs in governed contracts be private

### LINKED ISSUE #5931: Are privateArgs being revealed by being included in governance terms?
## What is the Problem Being Solved?

[@erights pointed out](https://github.com/Agoric/agoric-sdk/pull/5891#discussion_r942993457) that the privateArgs for the AMM are included in the terms of the AMM's governor.  That is apparently inadvertently revealing details (i.e.the poser invitation) that were intended to be closely held

## Description of the Design

Investigate this and figure out if this is acceptable (seems unlikely) or what to do about it.

## Security Considerations

It's all security.

## Test Plan

No plan until there's a theory on how to address this.

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
closes: #5931

## Description

[@erights pointed out](https://github.com/Agoric/agoric-sdk/pull/5891#discussion_r942993457) that the `privateArgs` for the AMM are included in the terms of the AMM's governor. That was revealing details (i.e.the poser invitation) that were intended to be closely held. This move the `privateArgs` of governed contracts from the governing contract's terms to a field of its own `privateArgs`.

### Security Considerations

This cleans up a confidentiality violation in governed contracts.

### Documentation Considerations

If we had more documentation on use of governed contracts, more cleanup would be required.

### Testing Considerations

sufficiently tested by existing unit tests and econ-behavior.

### CODE REVIEW

**Comment by turadg** [comment]
> If we had more documentation on use of governed contracts, more cleanup would be required.

🤣  
6q1usa

File: packages/governance/src/contractGovernor.js [hunk 1]
```diff
@@ -72,13 +72,14 @@ const validateQuestionFromCounter = async (zoe, electorate, voteCounter) => {
  * the Installation to be started, as well as an issuerKeywordRecord or terms
  * needed by the governed contract. Those details for the governed contract are
  * included in this contract's terms as a "governed" record. If the contract
- * expects privateArgs, those can be supplied as well.
+ * expects privateArgs, those passed in this contracts privateArgs under the
```
Comment (turadg) [inline packages/governance/src/contractGovernor.js]:
```suggestion
 * those will be provided in this contract's `privateArgs` under `.governed`.
```

**Review by turadg — APPROVED** [review]

File: packages/governance/src/contractGovernor.js (hunk 1 above)
Comment (Chris-Hibbert) [reply packages/governance/src/contractGovernor.js]: done

**Comment by erights** [comment]
> This cleans up a confidentiality violation in governed contracts.

This was leaking potentially powerful capabilities, not just information. It is therefore also an integrity violation.

**Comment by dckc** [comment]
@Mergifyio unqueue

**Comment by mhofman** [comment]
@mergifyio refresh

**Pull request merged**